src/rvm.ts

```diff
@@ -0,0 +1,148 @@
+      "Custom configure options, comma separated, double quote args that need quoting, default",
+  },
+];
+
+const interpreters = [
+  {
+    name: "rbx",
+    description: "Rubinius",
+    options,
+  },
+  {
+    name: "jruby",
+    description: "JRuby",
+    options: [...options],
+  },
+  {
+    name: "ruby",
+    description: "MRI/YARV Ruby (The Standard), defaults to 1.8.6",
+    options: [...options],
+  },
+  {
+    name: "ree",
+    description: "Ruby Enterprise Edition",
+    options: [...options],
+  },
+  {
+    name: "system",
+    description: "Use the system ruby (eg. pre-RVM state)",
+    options: [...options],
+  },
+  {
+    name: "default",
+    description: "Use RVM set default ruby and system if it hasn't been set",
+    options: [...options],
+  },
+];
+
+const completionSpec: Fig.Spec = {
+  name: "rvm",
+  description: "Ruby version manager",
+  options: [
+    {
+      name: ["--help", "-h"],
+      description: "Show help for rvm",
+      isPersistent: true,
+    },
+  ],
+  subcommands: [
+    {
+      name: "use",
+      description: "Setup current shell to use a specific ruby version",
+      subcommands: interpreters,
+    },
+    {
+      name: "reset",
+      description: "Remove default and current settings, exit the shell",
+      subcommands: [...interpreters],
+    },
+    {
+      name: "info",
+      description: "Show information about the current ruby environment",
+      subcommands: [...interpreters],
+    },
+    {
+      name: "list",
+      description: "List currently installed version",
+      subcommands: [...interpreters],
+    },
+    {
+      name: "reload",
+      description:
+        "Reload RVM source itself (useful after changing RVM source)",
+      subcommands: [...interpreters],
+    },
+    {
+      name: "implode",
+      description: "Remove all ruby installations it manages",
+      subcommands: [...interpreters],
+    },
+    {
+      name: "get",
+      description: "Upgrades RVM to the stable or git head branches",
+      subcommands: [...interpreters],
+    },
+    {
+      name: "do",
+      description: "Runs a named ruby file against specified and/or all rubies",
+      subcommands: [...interpreters],
+    },
+    {
+      name: "install",
+      description: "Install one or many ruby versions",
+      subcommands: [...interpreters],
+    },
+    {
+      name: "upgrade",
+      description:
+        "Install new ruby, copy gemsets, make gems pristine, remove old rubies",
+      subcommands: [...interpreters],
+    },
+    {
+      name: "reinstall",
+      description: "Remove ruby, install it, make gems pristine",
+      subcommands: [...interpreters],
+    },
+    {
+      name: "uninstall",
+      description: "Uninstall one or many ruby versions, leaves their sources",
+      subcommands: [...interpreters],
+    },
+    {
+      name: "remove",
+      description: "Remove one or many ruby versions, including their sources",
+      subcommands: [...interpreters],
+    },
```

Comment:
Interpreters are also valid main subcommands:
```suggestion
    },
    ...interpreters,
```